compile.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -e # Exit with nonzero exit code if anything fails
+
+make -C spec
+
+if [ -d out ]; then
+  echo Copy spec/index.html into out/index.html
+  cp spec/index.html out/index.html
+fi
```

Comment:
nit: not sure if we want newlines, but I generally prefer it because some editors will forcibly add it, causing spurious diffs

Reply:
Done